react-client/src/Components/App.js

```diff
@@ -59,6 +64,13 @@ function App() {
         });
 
         var eltarget = e.target;
+
+        var randomCode = "uniqueID"+uuidv4();
```

Comment:
`"uniqueID"` string is not necessary. remove it.

Reply:
noted already removed